bindings/python/tests/compiler_warning_bindings_roundtrip_test.py

```
@@ -0,0 +1,415 @@
+import depthai as dai
+import pytest
+
+
+def normalize(value):
```

Ruff (0.15.10)

[warning] Too many return statements (22 > 6)

(PLR0911)